functions-and-scope.js

```diff
@@ -16,6 +16,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+function cumLaude(array) {
+    let counter = 0;
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] >= 8) {
+            counter += 1;
```

Comment:
Dit kun je nog iets versimpelen
```suggestion
            counter++;
```